packages/jsii/lib/project-info.ts

```diff
@@ -142,6 +144,7 @@ export async function loadProjectInfo(projectRoot: string, { fixPeerDependencies
             && (pkg.contributors as any[]).map((contrib, index) => _toPerson(contrib, `contributors[${index}]`, 'contributor')),
 
     excludeTypescript: (pkg.jsii && pkg.jsii.excludeTypescript) || [],
+    exclude: (pkg.jsii && pkg.jsii.exclude) || [],
```

Comment:
Breaks surrounding style, and you might not want to fix them all up, so take it or leave it:

```suggestion
    exclude: pkg.jsii?.exclude ?? [],
```